gitlab-pipeline/stage/release.yml

```diff
@@ -12,6 +12,7 @@
   dependencies:
     - init:workspace
     - build:servers
+    - build:generate-delta-worker:docker
```

Comment:
sorry I missed that! there are just so many places and things to remember.

Reply:
It is common that we miss things in this area until we start the release process. Anyway, this is one pain point that will go away with the release process split :+1:

Reply:
prefect!